packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -27,9 +27,8 @@ import useBasket from '../commerce-api/hooks/useBasket'
 import Link from '../components/link'
 import RecommendedProducts from '../components/recommended-products'
 import {LockIcon} from '../components/icons'
-import {useVariationAttributes} from './'
 import {findImageGroupBy} from '../utils/image-groups-utils'
-import {useVariant} from './use-variant'
```

Comment:
The reason that I removed this hook is because I feel that contents of the modal shouldn't be derived from the state of the url path. That is find for the top level content aka the page, but in this case, it didn't work out because the hook lead to the incorrect product being used for finding images when on a product set page.